navigation/navigation-common/src/commonMain/kotlin/androidx/navigation/NavDestination.kt

```diff
@@ -102,6 +102,9 @@ public expect open class NavDestination(
     public open val displayName: String
         @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) get
 
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    public var id: Int
```

Comment:
I'd avoid exposing this to common
1. It doesn't make much sense for Compose
2. It messes up annotations on Android

I'd suggest to replace with expect/actual internal extension in tests